apps/src/code-studio/pd/application/teacher/ProfessionalLearningProgramRequirements.jsx

```diff
@@ -35,7 +35,7 @@ const getProgramName = program => {
     case PROGRAM_CSP:
       return 'CS Principles';
     case PROGRAM_CSA:
-      return 'CSA';
+      return 'Computer Science A';
     default:
       return 'CS Program';
```

Comment:
@dmcavoy If we're spelling out "Computer Science" for CSA, should we do it for CSD, CSP, and the default too?

Reply:
can do!